matlabCode/createBIDS_ieeg_electrodes_tsv.m

```diff
@@ -18,25 +20,38 @@
     '_electrodes.tsv']);
 
 
+
 %% make a participants table and save 
 
-% required columns
+
+
+%% required columns
+
 name = {''}; % Name of the electrode
 x = [0]';
 y = [0]';
 z = [0]';
 size = [0]'; % Diameter in mm 
 type = [0]'; % Type of intracranial electrode, one of [?surface?,  ?depth? , ?dbs?] 
 
-% optional columns:
+
+
+%% optional columns
+
 material = {''}; % Material of the electrodes
-tissue = {''}; % If a clinician has made an observation about the tissue underneath the electrode  (e.g. epilepsy, tumor, if nothing state n/a)
-manufacturer = {''}; % Optional field to specify the electrode manufacturer for each electrode. Can be used if electrodes were manufactured by more than one company.
-grid_size = {''}; % Optional field to specify the dimensions of the grid the electrode is a part of. Should be of the form `(M, N)`. E.g., (8, 8) or(1, 4).
 
-t = table(name,x,y,z,size,type,material,tissue,manufacturer,grid_size);
+tissue = {''}; % If a clinician has made an observation about the tissue 
+% underneath the electrode  (e.g. epilepsy, tumor, if nothing state n/a)
+
+manufacturer = {''}; % Optional field to specify the electrode manufacturer 
+% for each electrode. Can be used if electrodes were manufactured by more than one company.
 
-writetable(t,electrodes_tsv_name,'FileType','text','Delimiter','\t');
+grid_size = {''}; % Optional field to specify the dimensions of the grid the 
+% electrode is a part of. Should be of the form `(M, N)`. E.g., (8, 8) or(1, 4).
 
-clear size
```

Comment:
You need to keep 'clear size' after storing the table, since 'size' is also a Matlab function and it can give problems in case you don't clear it properly. Maybe all column variables should be cleaned here for real proper housekeeping, but this is an essential one.

Reply:
Shoot! I am stoopid! 
Thanks for spotting it!